pkgs/development/tools/misc/segger-jlink/default.nix

```diff
@@ -0,0 +1,118 @@
+{ lib
+, stdenv
+, fetchurl
+, autoPatchelfHook
+, makeWrapper
+, qt4
+, libudev
```

Comment:
libudev is an alias of udev, the library is in udev.dev, use udev and the .dev output should be used by nix for building

Reply:
Done and good to know. The `dev` output is automatically included for `buildInputs`, so it was a simple change.